pixels.go

```diff
@@ -0,0 +1,98 @@
+package waifu2x
+
+import (
+	"fmt"
+	"math"
+)
+
+type Pixels struct {
+	Width  int
+	Height int
+	Pix    []uint8 // 0以下を0, 255以上を255 とする
+}
+
+func NewPixels(width, height int) *Pixels {
+	return &Pixels{
+		Width:  width,
+		Height: height,
+		Pix:    make([]uint8, width*height),
+	}
+}
+
+func (p Pixels) NewImagePlane() *ImagePlane {
```

Comment:
__[gometalinter]__
* [golint] exported method Pixels.NewImagePlane should have comment or be unexported